src/CoreBundle/Resources/public/javascript/versionHistoryOverlay.js

```diff
@@ -0,0 +1,171 @@
+window.versionHistoryOverlayEventReceiver = undefined
```

Comment:
I originally created it exactly there but feedback from early review was to move it to other core scripts. I also think these should be in a single location within the change log bundle, will also make bundling and build additions easier.

Reply:
@umulmrum I was under the impression, that we use the bundle location for frontend assets - that cms assets will be placed at the same location is somewhat surprising to me.

Reply:
Moved.

Reply:
@rnuernberg I'm not aware that there is any distinction between frontend and backend assets. Aren't backend assets in that directory just by "accident" because almost all backend code is traditionally located in CoreBundle? I'm quite sure we already have backend JS in bundles under Resources/public.
Note that the backend assets are also available via `bundles` symlink.

But the important part is that we should not spread functionality across different bundles and should not introduce any code cycles (CoreBundle depending on code from CmsChangeLogBundle and vice versa). From this point of view I'm not happy with either way.

Maybe a cleaner way is to add the button as a plugin for CKeditor instead of cheating our way in (which I'd prefer anyway). We could try to decorate the `chameleon_system_core.wysiwyg.ckeditor_config_provider` to add the plugin.

Reply:
@umulmrum The button does not belong into the editor in the first place. I would have left it there for now since it was already added there and I wanted to avoid spending extra time. 

But bevor we start creating a plugin for the editor, we should instead move the history button to a more generic location next to the field value. That will also allow us to easily add the functionality for other fields in the feature.

Could you provide a suggestion where the button can be placed outside of the editor? My suggestion would be a third column for all fields [name][value][extras] - Then add the button into the newly generated column. @pixeljunkie any thoughts?

Reply:
Another screen for judgement: 
![selection_164](https://user-images.githubusercontent.com/958920/52577354-371e2900-2e22-11e9-949a-de20e25e1697.png)

Reply:
@umulmrum @ApricumMedia looks good to me. I would like the buttons all to be right or left aligned - not mixed.

Reply:
keep the button where it is for now.move the css/js back to where @umulmrum suggested.

Reply:
The client modules have already been moved back inside the Change Log Bundle, accessible via symlink `customer/web/bundles/chameleonsystemchangelog/javascript`.

I spent some time on the best spot for such a button. My first thought was to place it in the gigantic unused space underneath the field title (left-most column) to be consistent with the new function bar now also on the left side of all records in list views. However, this feature was only supposed to be built for the WYSIWYG editor, so I was working on that `TCMSField` and didn't have access to other columns from there.

My second thought was on the right side _on top of the wysiwyg editor_ but just placing it with the other buttons of the editor felt more natural.

If we decide on a completely new spot (making room) for this button, it should also be working for more buttons similar in function in the future and not be a one-off solution.